doc/releases/release-notes-2.4.rst

```diff
@@ -508,6 +508,12 @@ Libraries / Subsystems
 * Tracing:
   * Tracing backed API now checks if init function exists prio to calling it.
 
+* Shell:
+  * Fix case when number of arguments exceeded ``CONFIG_SHELL_ARGC_MAX``. Prior
+    2.3 when more arguments were provided then last argument contained remaining
+    arguments as one string. In 2.4 it was broken and system was crashing in
+    that case. Now it will report that command cannot be processed.
```

Comment:
s/Prior 2.3/Prior to 2.3/
s/2.4/2.3/

If you'd take a complete rewording:

```
* Change behavior when more than ``CONFIG_SHELL_ARGC_MAX`` arguments
  are passed.  Before 2.3 extra arguments were joined to the last
  argument.  In 2.3 extra arguments caused a fault.  Now the shell
  will report that the command cannot be processed.
```

Reply:
done